lib/internal/blob.js

```diff
@@ -85,7 +85,9 @@ const disallowedTypeCharacters = /[^\u{0020}-\u{007E}]/u;
 
 let ReadableStream;
 
-const enc = new TextEncoder();
+const { TextEncoder: UtilTextEncoder } = require('util');
+const enc = typeof TextEncoder !== 'undefined' ? new TextEncoder() : new UtilTextEncoder();
```

Comment:
It's likely just better to *just* use the import from `util` and not worry about checking to see if the global exists.